app/graphql/client.tsx

```diff
@@ -316,6 +333,67 @@ const MyUpdateSub = ({ children }: PropsWithChildren) => {
   return <LnUpdateHashPaidProvider value={lastHash}>{children}</LnUpdateHashPaidProvider>
 }
 
+const NetworkErrorToast: React.FC<{ networkError: ServerError | undefined }> = ({
```

Comment:
Is there a reason to include this component in this file? If not I think it would be cleaner to split it out.

Reply:
no particular reason. I have just put all the Apollo only thing into the client for code colocation convenience. but can move into a dedicated component.

Reply:
this has been update. could make sense to split up client more in different individual component but can do that in another PR